expertise/create_dataset.py

```diff
@@ -289,12 +290,23 @@ def retrieve_expertise(self):
 
     def get_submissions(self):
         invitation_ids = self.convert_to_list(self.config.get('paper_invitation', []))
+        paper_id = self.config.get('paper_id')
         submissions = []
 
         for invitation_id in invitation_ids:
             submissions.extend(list(openreview.tools.iterget_notes(
                 self.openreview_client, invitation=invitation_id)))
 
+        if paper_id:
+            submissions.append(self.openreview_client.get_note(paper_id))
+
+        # Bug: specter+mfr cannot handle a single submission
+        # Solution: create a copy of the note and modify the ID
+        if self.config.get('model') == 'specter+mfr' and len(submissions) == 1:
```

Comment:
What's the exact issue here @haroldrubio? Can we modify specter or mfr code to handle single submission?

Reply:
There was an error in the model code where something was being calculated incorrectly, specifically this assertion was failing:
https://github.com/openreview/openreview-expertise/blob/4e53d71b5fc33d61e36b488bfc788c4ef8faa198/expertise/models/multifacet_recommender/multifacet_recommender.py#L903

While you were away, we discussed it with Haw Shiuan and he agreed that we could just duplicate the submission as a simple fix.